### DISCUSSION TITLE
Progress buttons in setup overlay are broken

### DISCUSSION THREAD

**Discussion by Feodor0090** [ppy/osu#21500] [Q&A] [answered]
1) Click "get the tutorial"
2) Wait
3) Triangles color?
![изображение](https://user-images.githubusercontent.com/53872073/205345887-3cb627e7-98ef-4d4c-a5d3-27bea83f89b1.png)
4) Move you mouse out
5) Color reset to pink (green expected)
![изображение](https://user-images.githubusercontent.com/53872073/205346050-a2976aee-6740-4434-9888-6af910eb56f1.png)

The same with the second:
![изображение](https://user-images.githubusercontent.com/53872073/205346223-d6d52a04-fa09-4fb0-b0a0-23552be2978d.png)
![изображение](https://user-images.githubusercontent.com/53872073/205346262-3772eeb4-c298-4610-b921-16a6fff79e6b.png)

**Comment by peppy** [comment, marked as answer]
#21501
